src/arch/riscv/inc/arch/cpu.h

```diff
@@ -10,13 +10,22 @@
 
 extern cpuid_t CPU_MASTER;
 
+enum cpu_exts {
+    SSTC_EXTENSION,
+    EXTENSION_NOT_SUPPORTED
```

Comment:
I'd suggest all variants start wih the same prefix. For example:

```suggestion
    CPU_EXT_SSTC,
    CPU_EXT_NUM,
```